src/styles/templates/theme/margin.scss

```diff
@@ -0,0 +1,7 @@
+.mb4 {
```

Comment:
We should shift to use the padding and margin helpers for consistency.

Reply:
this was raised _prior_ to the helpers... i don't want to block so i'll probably do as a follow-up; is that ok with you?